build.sbt

```diff
@@ -20,6 +20,10 @@ commands := commands.value.filterNot { command =>
   }
 }
 
+addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll; docs/javafmtAll; +headerCreateAll")
+addCommandAlias("checkCodeStyle",
+  "+;headerCheckAll; scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll; docs/javafmtCheckAll")
```

Comment:
should this start `";+ headerCheckAll ` ?

Reply:
should starts with `;`

Reply:
And we can better introduce a plugin for this, @mdedetrich can you add these to `sbt-apache-sonatype`

Reply:
`"; +headerCheckAll` no, this is a bug that prevents the command from running and I had to fix it in other places.

> And we can better introduce a plugin for this, @mdedetrich can you add these to sbt-apache-sonatype

This does not belong in sbt-apache-sonatype, it would be a new sbt plugin that is specifically designed for pekko projects.